app/assets/javascripts/discourse/app/initializers/safari-13-deprecation.js

```diff
@@ -0,0 +1,30 @@
+import I18n from "I18n";
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+function setupMessage(api) {
+  const isSafari = navigator.vendor === "Apple Computer, Inc.";
```

Comment:
Can't you use the Safari detection from sniff-capabilities ?

Reply:
I tried, but on iOS 12, `isSafari` is false... so I guess our safari detection is broken for older versions. I started fixing it, but I figured it wasn't worth it since we're dropping support very soon anyway.